src/utils/slice.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright © 2020 The PES Open Source Team pesos@pes.edu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package utils
+
+func Contains(slc []string, item string) int {
+	for i, v := range slc {
+		if v == item {
+			return i
+		}
+	}
+	return -1
+}
```

Comment:
This entire thing could be replaced with a map right? it could go like this:
```go
set := map[string]bool{
  "1",  true,
  "2",  true,
}

if _, ok := set[ele]; ok {
...
}
```

It would be better than repeated loops I feel (although it is quite negligible here).

Reply:
Do you mean in allProcs, where I use this? In that case yes, I see now that I can replace it with a map like you mention. We could probably keep this function anyway, because I've found it to be generally useful in a few of my other golang projects (unless I've been doing all of them wrong :wink:)

If you meant using the map in the implementation of this function itself, then I'm not sure how you meant to use it

Reply:
Yep, the former is what I meant, the function seemed unnecessary. As for retaining it, it seems fine, but I'd like to not keep an unused function :P

We'll worry about whether to retain it or not later. Nevermind.

CC @MadhavJivrajani